deploy/rp-production-subscription.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "resources": [
+        {
+            "properties": {
+                "groupShortName": "rphealth",
+                "enabled": true
+            },
+            "name": "rphealth-ag",
+            "type": "Microsoft.Insights/ActionGroups",
+            "location": "Global",
+            "apiVersion": "2018-03-01"
+        }
+    ],
+    "outputs": {
+        "rpActionGroupId": {
+            "type": "string",
+            "value": "[resourceId('Microsoft.Insights/ActionGroups', 'rphealth-ag')]"
+        }
+    }
```

Comment:
don't think this is needed - you can deduce the name of the ActionGroup at the point where you need it.